test/integration/manifests/machineset.yaml

```diff
@@ -21,7 +21,7 @@ spec:
         sigs.k8s.io/cluster-api-machine-type: worker
         sigs.k8s.io/cluster-api-machineset: worker
     spec:
-      providerConfig:
+      providerSpec:
```

Comment:
Ha, this got forgotten. Please, remove this from the PR. The field can be renamed once we remove the `.providerConfig` field completely.